src/config.h

```diff
@@ -95,6 +95,9 @@ struct Config {
   // plain // /*
   int enableComments = 0;
 
+  // If true, filter and sort completion response.
+  bool filterAndSortCompletionResponse = true;
```

Comment:
I proposed to use a `struct ... completion`) because I expect there is need to have more options..

Reply:
We can do that later if need be, what other options do you think there would be?

Reply:
OK, I didn't know there are already many options residing in the completion namespace
```
  int includeCompletionMaximumPathLength = 30;
  // Whitelist that file paths will be tested against. If a file path does not
  // end in one of these values, it will not be considered for auto-completion.
  // An example value is { ".h", ".hpp" }
  //
  // This is significantly faster than using a regex.
  std::vector<std::string> includeCompletionWhitelistLiteralEnding;
  // Regex patterns to match include completion candidates against. They
  // receive the absolute file path.
  //
  // For example, to hide all files in a /CACHE/ folder, use ".*/CACHE/.*"
  std::vector<std::string> includeCompletionWhitelist;
  std::vector<std::string> includeCompletionBlacklist;
```